examples/patterns/grid/nested.html

```diff
@@ -3,29 +3,265 @@
 title: Grid / Nested columns
 category: _patterns
 ---
-<div grid-demo grid-outline>
+
+<div class="p-strip">
```

Comment:
We shouldn't add p-strip to examples as they are for copy and pasting for the component. There is no guarantee that the project has included strips.

Reply:
Fixed